### PR TITLE
Lesson 42

### PR DESCRIPTION
**Pull request by VladSmaug**
Was created new store from redux library and changed default store

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -3,7 +3,8 @@ import ReactDOM from "react-dom/client";
 import { BrowserRouter } from "react-router-dom";
 import reportWebVitals from "./reportWebVitals";
 
-import store from "./redux/state";
+// import store from "./redux/state";
```
Comment (Bohdan-Zavhorodskyi) [inline src/index.js:6]: Do we need this import?

File: src/redux/state.js [hunk 2]
```diff
@@ -1,100 +1,96 @@
...
-  },
-};
+//     this._callSubscriber(this._state);
+//   },
+// };
 
-export default store;
-window.store = store;
+// export default store;
+// window.store = store;
```
Comment (Bohdan-Zavhorodskyi) [inline src/redux/state.js:96]: Do we need all of this commented code?

**Review by Bohdan-Zavhorodskyi — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 1 above)
Comment (VladSmaug) [reply src/index.js:6]: for better understanding of what happens with state

File: src/redux/state.js (hunk 2 above)
Comment (VladSmaug) [reply src/redux/state.js:96]: for a better understanding of what happening

File: src/components/Dialogs/index.jsx [hunk 3]
```diff
@@ -12,6 +12,7 @@ import {
 
 const DialogItem = (props) => {
   const { name, id } = props;
+  console.log(name, "name");
```
Comment (Bohdan-Zavhorodskyi) [inline src/components/Dialogs/index.jsx]: Need to remove console.log

File: src/index.js (hunk 1 above)
Comment (Bohdan-Zavhorodskyi) [inline src/index.js:6]: Import is still here

File: src/components/Dialogs/index.jsx (hunk 3 above)
Comment (VladSmaug) [reply src/components/Dialogs/index.jsx]: Done!

**Review by Bohdan-Zavhorodskyi — APPROVED** [review]

**Pull request merged**